fun main (){
    throw Exception("Warning!!!!!")
}